smarts/core/traffic_history_provider.py

```diff
@@ -87,7 +87,11 @@ def teardown(self):
 
     @property
     def action_spaces(self) -> Set[ActionSpaceType]:
+        # What should be here?
+        # pytype: disable=bad-return-type
         return {}
+        # pytype: enable=bad-return-type
+
```

Comment:
Another larping dictionary.

```suggestion
        return set()
```